jni/iri-pearldiver-exlib.c

```diff
@@ -27,7 +28,7 @@ Java_com_iota_iri_hash_PearlDiver_exlib_1search(JNIEnv *env,
         return JNI_FALSE;
     /****************************************************************/
 
-    int8_t *result = dcurl_entry(arg_trytes->data, mwm);
+    int8_t *result = dcurl_entry(arg_trytes->data, mwm, threads);
```

Comment:
What if `threads` equals to `0`?

Reply:
Will IRI be functioned unexpectedly?

Reply:
Nope.

If the `--pow-threads` option is not used, the default value of **IRI** thread number would be `0`.
In that case, the available thread number would be automatically detected by **dcurl**.

Possible value:
- Max core number - 1
- Environment variable `DCURL_NUM_CPU` ( <= max core number - 1 )

Reply:
However, I forgot to assign the value detected by `dcurl`.

Reply:
Done.